enhancements/olm/platform-operators.md

```diff
@@ -0,0 +1,609 @@
+Using an [admission controller](#admission-controller-for-package-name-uniqueness) to validate this package name uniqueness is listed as an alternative for this proposal. Future phases may re-evaluate this implementation and address any quality of life improvements.
+
+### FeatureGate
+
+The components outlined in the [Proposal](#Proposal) section will be delivered as a tech preview feature for openshift clusters, and therefore this mechanism won't be installed by default.
+
+An admin can enable the "TechPreviewNoUpgrade" feature set before cluster creation as outlined in the [Workflow Description](#Workflow-Description) section:
+
+> Note: admins can only enable POs through the TechPreviewNoUpgrade feature set, and therefore enabling the CustomNoUpgrade feature set requirement performing some manual steps.
+
+```yaml
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade
+```
+
+Enabling the tech preview feature set will result in the installation of the PO components in the cluster. This requires adding the "release.openshift.io/feature-gate" annotation key to the PO's downstrema CVO manifests, and setting that key's value to "TechPreviewNoUpgrade".
+
+#### Reconcile POM "aggregate" ClusterOperator object
+
+The PO manager will be responsible for acting as a proxy between the CVO, and the underlying PO packages during phase 0. This can be done by reconciling an "aggregate" ClusterOperator that reflects the overall state of installing PO packages.
+Future phases may require PO packages to reconcile OLM's OperatorCondition API, or the directly manage a ClusterOperator resource to express their health and overall state.
+
+> Note: The plain provisioner has the current limitation around applying Bundle contents that will produce runtime issues. In the case that unpacked Bundle contents contain a Deployment manifest, and that manifest references an invalid container image, then the result would be an unavailable container process after being applied to the cluster.
+Despite the presence of runtime issues here, the BundleDeployment that manages those manifests would reflect a successful installation state. Addressing this limitation is out-of-scope for phase 0, and will be addressed in future phases.
+
+A separate controller will be introduced that will watch the "aggregate" ClusterOperator resource, and watch for any BundleDeployment events in the cluster.
+During reconciliation, this controller will be responsible for listing the BundleDeployments in the cluster, aggregating the installation state of those individual resources into a single status, and writing that status back to the ClusterOperator resource that controller is managing.
```

Comment:
```suggestion
During reconciliation, this controller will be responsible for listing the BundleDeployments in the cluster owned by PlatformOperators, aggregating the installation state of those individual resources into a single status, and writing that status back to the ClusterOperator resource that controller is managing.
```

Reply:
Updated in https://github.com/openshift/enhancements/pull/1170/commits/bc14b4537125d43153cc452cd686e02cf96630b2.